test/integration/default/local_alb.rb

```diff
@@ -0,0 +1,60 @@
+require 'awspec'
+puts $:
```

Comment:
What does this do? :)

Reply:
I've deleted the puts. It was debugging info.